examples/http-redirects/http_redirects.bal

```diff
@@ -4,21 +4,56 @@ import ballerina/log;
 import ballerina/mime;
 
 endpoint http:Client clientEP {
-    url: "http://www.mocky.io",
+    url: "http://localhost:9090",
     followRedirects: { enabled: true, maxCount: 5 }
 };
 
 function main(string... args) {
 
     // Send a GET request to the specified endpoint.
-    var returnResult = clientEP->get("/v2/59d590762700000a049cd694");
+    var returnResult = clientEP->get("/redirect1");
     match returnResult {
```

Comment:
There should be a newline between, L14 and L15 to render the comment line only to L14